### ISSUE TITLE
Unrecognized C ABI (linux glibc 2.32.0)

### ISSUE THREAD

**Issue by darltrash** [ziglang/zig#6192]
When i try to compile anything that has C imports, i get the error:
```
Build Dependencies...start [2] Compile C Objects [1/1] start.S...Unrecognized glibc version: 2.32.0
unrecognized C ABI
tetris...The following command exited with error code 1:
/usr/bin/zig build-exe /home/USERNAME/tetris/src/main.zig --c-source -std=c99 /home/USERNAME/tetris/stb_image-2.22/stb_image_impl.c --library c --library glfw --library epoxy --cache-dir /home/USERNAME/tetris/zig-cache --name tetris -I /home/USERNAME/tetris/stb_image-2.22 --cache on 

Build failed. The following command failed:
/home/USERNAME/tetris/zig-cache/o/oL7mttLLH2knM0DHODbyYiQqAfUtUPHxHllNfa5NPNV94yLrkaiaxb5bkP66d9xn/build /usr/bin/zig /home/USERNAME/tetris /home/USERNAME/tetris/zig-cache
```
(i got this error code while running `zig build` in [this repo](https://github.com/andrewrk/tetris))

OS: Arch Linux (latest)
GlibC: 2.32.0
Architecture: x86_64
Zig version: 0.6.0 (also happens with master)

**Comment by Jan200101** [comment]
should be dealt with by #6166

**Comment by darltrash** [comment]
Still the exact same error.

**Comment by sorenbug** [comment]
@darltrash Are you on the latest Zig? It works for me.

**Comment by darltrash** [comment]
> @darltrash Are you on the latest Zig? It works for me.

yes, i am in the latest version, i used the [zig-git](https://aur.archlinux.org/packages/zig-git/) package

**Comment by justcosmic** [comment]
I encountered the same problem with zig 0.60.0 and glibc 2.32.0. However, I tested the lastest version from zig-git package and tetris compiles and runs just fine now.

**Comment by Jan200101** [comment]
It was merged into master with #6212

**Comment by andrewrk** [comment]
Note also that after #6250 when this happens next time (when glibc 2.33 is released), zig will provide glibc 2.32 until 2.33 support is added, so everything will continue working.

**Comment by freswa** [comment]
We'd like to package zig for arch, but there is currently no zig release compatible to glibc 2.32. Is there any plan to release this patch as 0.6.1?

**Comment by Jan200101** [comment]
support for glibc 2.32 in Zig will be released with 0.7.0
